spring-cloud-dataflow-common/spring-cloud-dataflow-common-test-docker/src/test/java/org/springframework/cloud/dataflow/common/test/docker/compose/execution/CommandTests.java

```diff
@@ -20,88 +20,80 @@
+    private ErrorHandler errorHandler = mock(ErrorHandler.class);
     private Command dockerComposeCommand;
     private final List<String> consumedLogLines = new ArrayList<>();
     private final Consumer<String> logConsumer = s -> consumedLogLines.add(s);
 
-    @Before
-    public void before() throws IOException {
+    @BeforeEach
+    void prepareForTest() throws IOException {
+        when(dockerComposeExecutable.commandName()).thenReturn("docker-compose");
         when(dockerComposeExecutable.execute(any())).thenReturn(executedProcess);
+        when(dockerComposeExecutable.execute(any(String[].class))).thenReturn(executedProcess);
         dockerComposeCommand = new Command(dockerComposeExecutable, logConsumer);
-
         givenTheUnderlyingProcessHasOutput("");
         givenTheUnderlyingProcessTerminatesWithAnExitCodeOf(0);
     }
 
-    @Test public void
-    invoke_error_handler_when_exit_code_of_the_executed_process_is_non_0() throws IOException, InterruptedException {
+    @Test
+    void invokeErrorHandlerWhenExitCodeOfTheExecutedProcessIsNonZero() throws Exception {
         int expectedExitCode = 1;
         givenTheUnderlyingProcessTerminatesWithAnExitCodeOf(expectedExitCode);
         dockerComposeCommand.execute(errorHandler, "rm", "-f");
-
         verify(errorHandler).handle(expectedExitCode, "", "docker-compose", "rm", "-f");
     }
 
-    @Test public void
-    not_invoke_error_handler_when_exit_code_of_the_executed_process_is_0() throws IOException, InterruptedException {
+    @Test
+    void notInvokeErrorHandlerWhenExitCodeOfTheExecutedProcessIsZero() throws Exception {
         dockerComposeCommand.execute(errorHandler, "rm", "-f");
-
         verifyNoMoreInteractions(errorHandler);
     }
 
-    @Test public void
-    return_output_when_exit_code_of_the_executed_process_is_non_0() throws IOException, InterruptedException {
+    @Test
+    void returnOutputWhenExitCodeOfTheExecutedProcessIsNonZero() throws Exception {
         String expectedOutput = "test output";
         givenTheUnderlyingProcessTerminatesWithAnExitCodeOf(1);
         givenTheUnderlyingProcessHasOutput(expectedOutput);
         String commandOutput = dockerComposeCommand.execute(errorHandler, "rm", "-f");
-
         assertThat(commandOutput, is(expectedOutput));
     }
 
-    @Test public void
-    return_output_when_exit_code_of_the_executed_process_is_0() throws IOException, InterruptedException {
+    @Test
+    void returnOutputWhenExitCodeOfTheExecutedProcessIsZero() throws Exception {
         String expectedOutput = "test output";
         givenTheUnderlyingProcessHasOutput(expectedOutput);
         String commandOutput = dockerComposeCommand.execute(errorHandler, "rm", "-f");
-
         assertThat(commandOutput, is(expectedOutput));
     }
 
-    @Test public void
-    give_the_output_to_the_specified_consumer_as_it_is_available() throws IOException, InterruptedException {
```

Comment:
Because this is a test and nobody is going to do anything w/ the annoying checked exception (nobody cares at this point) it is just clutter to list them out IMO. To me it provides no value - only clutter.